lib/archethic_web/live/top_transactions_component.ex

```diff
@@ -0,0 +1,156 @@
+defmodule ArchethicWeb.ExplorerIndexLive.TopTransactionsComponent do
+  @moduledoc """
+    Live component for Dashboard Explorer to display last 10 transactions
+  """
+
+  use ArchethicWeb, :live_component
+
+  alias Archethic.BeaconChain
+
+  alias Archethic.Election
+
+  alias Archethic.P2P
+  alias Archethic.P2P.Node
+  alias Archethic.P2P.Message.GetCurrentSummaries
+  alias Archethic.P2P.Message.TransactionSummaryList
+
+  alias ArchethicWeb.ExplorerLive.TopTransactionsCache
+
+  def mount(socket) do
+    socket =
+      socket
+      |> assign(:transactions, [])
+
+    {:ok, socket}
+  end
+
+  def update(
+        %{transaction: transaction} = _assigns,
+        socket
+      )
+      when not is_nil(transaction) do
+    {:ok, transactions} =
+      TopTransactionsCache.resolve_put(transaction, fn ->
```

Comment:
Why are you fetching the list if you are receiving a new transaction, this part of fetching should be at the initialization or preloading of the component.